oceanstream/L3_regridded_data/mvbs_computation.py

```diff
@@ -0,0 +1,77 @@
+from typing import Literal, Optional
+
+import echopype as ep
```

Comment:
Maybe it would be better to import only the needed functions:
`from echopype.commongrid.api import compute_MVBS, compute_MVBS_index_binning`